libkbfs/block_tree.go

```diff
@@ -269,6 +271,24 @@ func (bt *blockTree) getNextDirtyBlockAtOffset(ctx context.Context,
 	return ptr, parentBlocks, block, nextBlockStartOff, startOff, nil
 }
 
+// TODO: rename these structs
+// structs used for passing data to and from GetBlocksForOffsetRange tasks
+type task struct {
+	ptr        BlockPointer
+	pblock     *BlockWithPtrs
```

Comment:
`BlockWithPtrs` is an interface, and can be `nil` on its own, so no need to have a pointer to it here.